administrator/components/com_redirect/Model/LinksModel.php

```diff
@@ -204,20 +213,21 @@ public function batchProcess($batch_urls)
 		$db    = $this->getDbo();
 		$query = $db->getQuery(true);
 
-		$params = ComponentHelper::getParams('com_redirect');
-		$state  = (int) $params->get('defaultImportState', 0);
+		$params  = ComponentHelper::getParams('com_redirect');
+		$state   = (int) $params->get('defaultImportState', 0);
+		$created = Factory::getDate()->toSql();
 
-		$columns = array(
+		$columns = [
 			$db->quoteName('old_url'),
 			$db->quoteName('new_url'),
 			$db->quoteName('referer'),
 			$db->quoteName('comment'),
 			$db->quoteName('hits'),
 			$db->quoteName('published'),
 			$db->quoteName('created_date')
-		);
+		];
 
-		$query->columns($columns);
+		$i = 0;
 
 		foreach ($batch_urls as $batch_url)
```

Comment:
```suggestion
		foreach ($batch_urls as $i => $batch_url)
```
Use the array index without having to create an extra variable/counter.

Reply:
thanks